src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -344,12 +344,12 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_expm1)
 
 // cos
 MXNET_OPERATOR_REGISTER_UNARY(cos)
-.describe(R"code(Cosine, element-wise.
+.describe(R"code(Returns element-wise trigonometric cosine value of the input.
 
-The input is in radians (:math:`2\pi` rad equals 360 degress).
+The input is in radians (:math:`2\pi` rad equals 360 degrees).
 
 .. math::
-   cos([0, \pi/4, \pi/2]) = [1, 0.707, 0]
+   cos([0, \pi/4, \pi/2]) = [1, 0.707,  -4.37113883e-08]
```

Comment:
it mainly explains this function accepts radians rather than degrees. 

i suggest not using a real value such as ` -4.37113883e-08`, a fake value such as `0` makes the doc shorter